.gitignore

```diff
@@ -15,9 +15,15 @@ models/**
 .coverage
 *.pyc
 *~
+*.swp
+.python-version
```

Comment:
What does python-version do?

Reply:
it is a hidden file from `pyenv` that contains that information of which python version should be used in the project out of all the python version available